### ISSUE TITLE
Assign plugin functionality to a certain part of the website

### ISSUE THREAD

**Issue by alberto2000** [lukehaas/Scrollify#51]
Why not implement the plugin following the convention of $('#desired-container').scrollify({section: 'desired-section-element'}); so it can be assigned to only specific parts of the website? I have a one page layout but it is two-parted (50/50 left and right) and I only want the right part to have this functionality - when I apply it as it is now, only the hash changes but no scroll effect whatsoever...

**Comment by lukehaas** [comment]
You might be able to achieve what you want by using the 'target' option.

**Comment by altmind** [comment]
`target` is not documented, telling from source, it seems that it defines the (overflow:visible) element in which the conentent will scroll. which may be not what @alberto2000 want.

actually, i think i have the same necessity as @alberto2000 , i need to have half-of-screen of content on page and snappable blocks below, so snappable behavior starts from specific height. haven't found a solution yet.

**Comment by lukehaas** [comment]
@altmind yeah, when I made that comment I forgot that `target` was still a developmental feature.
This functionality won't be coming to Scrollify any time soon. It may be best to look for an alternative plugin for now.

**Comment by bini7a** [comment]
@alberto2000 You can manage them defining different section name. For left part define all section name to 'left-section' and for right part define all section name to 'right-section' and call scrollify for both.

**Comment by lukehaas** [comment]
@bini7a That won't work, Scrollify scrolls the window, not the document.

**Comment by bini7a** [comment]
@lukehaas I thought both part has scroll bar. Doesn't that work even?

**Comment by drewbaker** [comment]
+1 to allow containers other than `window`

**Comment by lukehaas** [comment]
I'm happy for this feature to be added to Scrollify if anyone wants to contribute.

**Comment by alberto2000** [comment]
I'd be glad to contribute - @lukehaas if you could give me some indication on which parts may need refactoring or just a short crash course into the architecture of the plugin? From very first telling I'd just go an replace all references to $window with the new, configurable target element and go from there?

**Comment by lukehaas** [comment]
@alberto2000 It could be as straight forward as that. 
In Scrollify currently mouse scroll events are attached to the window and touch events are attached to the document, (can't remember why the difference now). The scroll animation is applied to "html,body". 
Getting all those to point to a configurable reference is the best place to start.

**Comment by alberto2000** [comment]
On it, will make a PR asap

**Comment by alberto2000** [comment]
PR submitted https://github.com/lukehaas/Scrollify/pull/158

**Comment by lukehaas** [comment]
@alberto2000 thanks I'll take a look soon.

**Comment by alberto2000** [comment]
@lukehaas what about https://github.com/lukehaas/Scrollify/pull/158 - also it'd be good if some could test it and report any feedback
